docs/private-networks/how-to/backup.md

```diff
@@ -42,6 +42,13 @@ If log messages signify a corrupt database, the cleanest way to recover is:
 1. Restore the data from a [previous backup](#data-backups).
 1. Restart the node.
 
+## Prevent accidental downgrade
+
+When restarting Besu, accidentally using an earlier version of Besu might risk corrupting your database.
+To protect against incompatibility between versions, set the
+[`--version-compatibility-protection`](../../public-networks/reference/cli/options.md#version-compatibility-protection)
+option to `true`.
```

Comment:
It's probably worth adding into this description that the option defaults to `true` for any non-named network. So if you're connecting to a public chain such as `goerli` it won't be enabled, but if you're using your own genesis file it will be enabled. You can force it on or off by setting the command yourself.